main.go

```diff
@@ -27,83 +28,44 @@ import (
 func main() {
 	log.Namespace = "dp-frontend-router"
 
-	if v := os.Getenv("BIND_ADDR"); len(v) > 0 {
-		config.BindAddr = v
-	}
-	if v := os.Getenv("BABBAGE_URL"); len(v) > 0 {
-		config.BabbageURL = v
-	}
-	if v := os.Getenv("RENDERER_URL"); len(v) > 0 {
-		config.RendererURL = v
-	}
-	if v := os.Getenv("DATASET_CONTROLLER_URL"); len(v) > 0 {
-		config.DatasetControllerURL = v
-	}
-	if v := os.Getenv("FILTER_DATASET_CONTROLLER_URL"); len(v) > 0 {
-		config.FilterDatasetControllerURL = v
-	}
-	if v := os.Getenv("GEOGRAPHY_CONTROLLER_URL"); len(v) > 0 {
-		config.GeographyControllerURL = v
-	}
-	if v := os.Getenv("ZEBEDEE_URL"); len(v) > 0 {
-		config.ZebedeeURL = v
-	}
-	if v := os.Getenv("DOWNLOADER_URL"); len(v) > 0 {
-		config.DownloaderURL = v
-	}
-	if v := os.Getenv("PATTERN_LIBRARY_ASSETS_PATH"); len(v) > 0 {
-		config.PatternLibraryAssetsPath = v
-	}
-	if v := os.Getenv("SITE_DOMAIN"); len(v) > 0 {
-		config.SiteDomain = v
-	}
-
-	if v := os.Getenv("REDIRECT_SECRET"); len(v) > 0 {
-		config.RedirectSecret = v
-	}
-
-	if v := os.Getenv("ANALYTICS_SQS_URL"); len(v) > 0 {
-		config.SQSAnalyticsURL = v
+	cfg, err := config.Get()
+	ctx := context.Background()
+	if err != nil {
+		log.Event(ctx, "unable to retrieve service configuration", log.Error(err))
+		os.Exit(1)
 	}
 
-	if v := os.Getenv("CONTENT_TYPE_BYTE_LIMIT"); len(v) > 0 {
-		a, err := strconv.Atoi(v)
-		if err == nil {
-			config.ContentTypeByteLimit = int(a)
-		}
-	}
+	fmt.Println("dataset", cfg.DatasetRoutesEnabled)
```

Comment:
Remove - or replay with `log.Event` if it is useful

Reply:
Removed